[tests] Work around sqlserver 2022-latest latest image breaking with testcontainers

radical opened the pull request:
[tests] Workaround sqlserver 2022-latest latest image failing

.. with testcontainers because the path `/opt/mssql-tools/bin/sqlcmd`
changed to `/opt/mssql-tools18/bin/sqlcmd`. But this hardcoded path is
being used in testcontainers, causing it to fail.

https://github.com/testcontainers/testcontainers-dotnet/issues/1220
https://github.com/microsoft/mssql-docker/issues/892

Issue: https://github.com/dotnet/aspire/issues/5057

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/5058)

joperezr commented on tests/Aspire.Microsoft.EntityFrameworkCore.SqlServer.Tests/Aspire.Microsoft.EntityFrameworkCore.SqlServer.Tests.csproj:
```diff
@@ -3,6 +3,9 @@
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <RunTestsOnHelix>true</RunTestsOnHelix>
+
+    <!-- ActiveIssue: https://github.com/dotnet/aspire/issues/5057 -->
```
I think we also need it for Aspire.Microsoft.Data.SqlClient.Tests as those are also failing.

joperezr reviewed (approved):
Apart from also adding the other sqlclient skip tests, this looks good to me. Thanks @radical

radical committed:
[tests] Workaround sqlserver 2022-latest latest image failing

.. with testcontainers because the path `/opt/mssql-tools/bin/sqlcmd`
changed to `/opt/mssql-tools18/bin/sqlcmd`. But this hardcoded path is
being used in testcontainers, causing it to fail.

https://github.com/testcontainers/testcontainers-dotnet/issues/1220
https://github.com/microsoft/mssql-docker/issues/892

radical committed:
add comment referencing the issue

eerhardt reviewed (approved):
LGTM. Nice find!
